<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sfcGUI</class>
 <widget class="QMainWindow" name="sfcGUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surface Creator from Point Cloud</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#1d1d1d;
}

QWidget{
background-color:#3d3d3d;
color:#fff;
}

QMenuBar{
background-color:#1d1d1d;
padding:5px;
font: 10pt &quot;Times New Roman&quot;;
}

QMenuBar::item{
background-color:#1d1d1d;
color:#fff;
padding-right:10px;
padding-left:10px;
}

QMenuBar::item:selected{
background-color:#9d948c;
color:#fff;
}

QMenu{
background-color:#3d3d3d;
color:#fff;
}

QMenu::item:selected{
background-color:#9d948c;
}

QStatusBar{
color:#fff;
background-color:#1d1d1d;
}

QPushButton{
border-style:solid;
background-color:#a6a6a6;
color:#fff;
border-radius:5px;
}

QPushButton:hover{
color:#ccc;
background-color: qlineargradient(spread:pad, x:1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(45,45,45,255), stop:0.505682 rgba(45,45,45,255), stop:1 rgba(29,29,29,255));
border-color:#29d1d9;
border-width:2px;
}

QPushButton:pressed{
background-color:  qlineargradient(spread:pad, x:1:0.517, y1:0, x2:0.517, y2:1, stop:0 rgba(29,29,29,255), stop:0.505682 rgba(45,45,45,255), stop:1 rgba(29,29,29,255));
}

QSpinBox{
border-style:solid;
background-color:#a6a6a6;
color:#fff;
border-radius:4px;
}

QComboBox{
border-style:solid;
background-color:#a6a6a6;
color:#fff;
border-radius:4px;
}

QStatusBar QLCDNumber{
border: 1px solid red;
border-radius: 3px;
}

QStatusBar QLabel{
color:#fff;
padding-left:5px;
font: 8pt &quot;Times New Roman&quot;;
}

</string>
  </property>
  <widget class="QWidget" name="central">
   <widget class="GLRender" name="glrender">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="operations" native="true">
    <property name="geometry">
     <rect>
      <x>551</x>
      <y>-3</y>
      <width>261</width>
      <height>601</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>231</width>
       <height>591</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="rightSide">
      <item>
       <widget class="QGroupBox" name="details">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>Computations capacity</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>101</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Processor Cores</string>
         </property>
        </widget>
        <widget class="QNSpinBox" name="core">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>20</y>
           <width>42</width>
           <height>16</height>
          </rect>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="accelerated">
          <bool>false</bool>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>20</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
        <widget class="QPushButton" name="compButton">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QPushButton" name="surf">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Surface</string>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="draw">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>75</height>
         </size>
        </property>
        <property name="title">
         <string>Display</string>
        </property>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>20</y>
           <width>47</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>211</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="StepsOnTree">
          <item>
           <widget class="QPushButton" name="left_child">
            <property name="text">
             <string>Left child</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="right_child">
            <property name="text">
             <string>Right child</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="parent">
            <property name="text">
             <string>Parent</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QComboBox" name="display">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>Points</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Box</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Surface</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>All</string>
          </property>
         </item>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="filterBox">
        <property name="title">
         <string>Filters</string>
        </property>
        <widget class="QScrollArea" name="scrollArea">
         <property name="geometry">
          <rect>
           <x>-10</x>
           <y>12</y>
           <width>241</width>
           <height>241</height>
          </rect>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>222</width>
            <height>248</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolBox" name="filters">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>230</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>250</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="tabSpacing">
              <number>5</number>
             </property>
             <widget class="QWidget" name="page">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>204</width>
                <height>204</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Filter1</string>
              </attribute>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="surfaceBox">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>150</height>
         </size>
        </property>
        <property name="title">
         <string>Surface reconstructor</string>
        </property>
        <widget class="QToolBox" name="surfacerec">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>211</width>
           <height>111</height>
          </rect>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>211</width>
            <height>84</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Page 2</string>
          </attribute>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="performance" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>580</y>
      <width>551</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="sadsadsa">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>229</width>
      <height>60</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>60</height>
     </size>
    </property>
    <property name="title">
     <string>j</string>
    </property>
    <widget class="QSlider" name="max_slider">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="max_poi">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Poi in 1 box</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionOutlier_removals"/>
    <addaction name="separator"/>
    <addaction name="actionCGAL_based_outl"/>
    <addaction name="actionPlane_base_out"/>
    <addaction name="separator"/>
    <addaction name="actionSimplifiers"/>
    <addaction name="separator"/>
    <addaction name="actionGrid_mine"/>
    <addaction name="actionCGAL_based_Grid"/>
    <addaction name="actionCGAL_based_Hieararchy"/>
    <addaction name="separator"/>
    <addaction name="actionSmoothers"/>
    <addaction name="separator"/>
    <addaction name="actionCGAL_based_Jet_smoother"/>
   </widget>
   <widget class="QMenu" name="menuSurface">
    <property name="title">
     <string>Surface</string>
    </property>
    <addaction name="actionCGAL_based_Poisson"/>
    <addaction name="actionVoronoi_based"/>
    <addaction name="actionPartial_diff"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilter"/>
   <addaction name="menuSurface"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOutlier_removals">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Outlier removals</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actionCGAL_based_outl">
   <property name="text">
    <string>CGAL based</string>
   </property>
  </action>
  <action name="actionPlane_base_out">
   <property name="text">
    <string>Plane based</string>
   </property>
  </action>
  <action name="actionSimplifiers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Simplifiers</string>
   </property>
  </action>
  <action name="actionGrid_mine">
   <property name="text">
    <string>Grid mine</string>
   </property>
  </action>
  <action name="actionCGAL_based_Grid">
   <property name="text">
    <string>CGAL based Grid</string>
   </property>
  </action>
  <action name="actionCGAL_based_Hieararchy">
   <property name="text">
    <string>CGAL based Hieararchy</string>
   </property>
  </action>
  <action name="actionSmoothers">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Smoothers</string>
   </property>
  </action>
  <action name="actionCGAL_based_Jet_smoother">
   <property name="text">
    <string>CGAL based Jet smoother</string>
   </property>
  </action>
  <action name="actionCGAL_based_Poisson">
   <property name="text">
    <string>CGAL based Poisson</string>
   </property>
  </action>
  <action name="actionVoronoi_based">
   <property name="text">
    <string>Voronoi based</string>
   </property>
  </action>
  <action name="actionPartial_diff">
   <property name="text">
    <string>Partial diff.</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLRender</class>
   <extends>QOpenGLWidget</extends>
   <header>glrender.h</header>
  </customwidget>
  <customwidget>
   <class>QNSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qnspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/sfc.qrc"/>
 </resources>
 <connections/>
</ui>
